src/main/scala/mesosphere/marathon/core/groupmanager/package.scala

```diff
@@ -0,0 +1,6 @@
+package mesosphere.marathon.core
+
+/**
+  * This package allows to manage all group related actions.
+  */
+package object groupmanager
```

Comment:
Sorry, I just followed the pattern of some other modules in `core` without giving it too much thought. Better remove that file then?